modules/ROOT/pages/database-administration/aliases/remote-database-alias-configuration.adoc

```diff
@@ -131,92 +134,232 @@ dbms.security.key.name=key-name
 
 Where `password.sh` might look like this:
 
-[source, Password.sh]
+.Password.sh
+[source, bash]
 ----
 #!/bin/bash
 echo "$KEYSTORE_PASSWORD_ENVIRONMENT_VARIABLE"
 ----
 
-Additionally, don’t forget to change the permissions of the configuration file, and start Neo4j with the command expansion flag:
+Additionally, do not forget to change the permissions of the configuration file and start Neo4j with the command expansion flag:
 
-[source]
+[source, bash]
 ----
 chmod 640 conf/neo4j.conf
 bin/neo4j start --expand-commands
 ----
 
-== Manage remote database aliases
+=== Manage remote database aliases
```

Comment:
This section is about  Carol, isn’t it? When she gets all privileges, she can manage the remote database alias

Reply:
I think so.